proto/streamlit/proto/Metric.proto

```diff
@@ -29,6 +29,7 @@ message Metric {
   MetricColor color = 5;
   string help = 6;
   LabelVisibilityMessage label_visibility = 7;
+  bool border = 8;
```

Comment:
Since this a boolean variable, I would name it something like `show_border` or something like this, also in the code. This variable name sounds more like the border definition itself.